backend/settings/base.py

```diff
@@ -131,15 +144,17 @@
 # Users avatar storage
 # https://cloudinary.com/documentation/django_integration
 cloudinary.config(
```

Comment:
We don't need cloudinary anymore, right?

Reply:
Yea, I'll remove all its configurations. It's also in models though, can I switch the field back to ImageField? (I'll find out the right attribute name)

Reply:
@Del-sama please, suggestions for this?

Reply:
I'm guessing we can use an ImageField and update with a new Github image if the user updates the image. That, or we can just remove the field and load the image directly from Github. We can cache the image and try to update the cache each time the user logs in, but this will be assuming every user has a Github image. Another alternative is to by default use whatever image they uploaded on their Github/Gitlab but give them the option to update the image. I'm guessing that option requires the `CloudinaryField`. What do you think @kosyfrances

Reply:
how about I use the default models.ImageField attribute with its default settings and then open a new PR for customization?

Reply:
Remember that Github gives you the URL to the person's avatar. This means, we are not storing the image, but the URL,  which is a string. So you do not need an image field.